lib/vanagon/project.rb

```diff
@@ -665,6 +669,27 @@ def save_manifest_json
       end
     end
 
+    # Writes a yaml file at `output/<name>-<version>.<platform>.settings.yaml`
+    # containing settings used to build the current project on the platform
+    # provided (and a corresponding sha1sum file) if `yaml_settings` has been
+    # set in the projcet definition.
+    #
+    # @param [Vanagon::Platform] the platform to publish settings for
+    def publish_yaml_settings(platform)
+      return unless @yaml_settings
+
+      filename = "#{@name}-#{@version}.#{platform.name}.settings.yaml"
+      filepath = File.join('output', filename)
+
+      File.open(filepath, 'w') do |f|
+        f.write(@settings.to_yaml)
+      end
+
+      File.open("#{filepath}.sha1", 'w') do |f|
+        f.write(%x(sha1sum #{filepath}))
```

Comment:
you're going to need to use `platform.shasum` here to make sure you're calling the right command